lib/entities/user-ui-config.ts

```diff
@@ -0,0 +1,54 @@
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import copy from 'fast-copy'
+import { BasicMetaSysProps, DefaultElements, MakeRequest, SysLink } from '../common-types'
+import createUserUIConfigApi from '../create-user-ui-config-api'
+import enhanceWithMethods from '../enhance-with-methods'
+
+export type UserUIConfigProps = {
+  /**
+   * System metadata
+   */
+  sys: UserUIConfigSysProps
+
+  assetListViews: ViewFolder[]
+  entryListViews: ViewFolder[]
+}
+
+export interface UserUIConfigSysProps extends BasicMetaSysProps {
+  space: SysLink
+  environment: SysLink
+}
+
+interface ViewFolder {
+  id: string
+  title: string
+  views: View[]
+}
+
+interface View {
+  id: string
+  title: string
+  order?: {
+    fieldId: string
+    direction: 'ascending' | 'descending'
+  }
+  page?: number
```

Comment:
These have to go, no? Or was it the page property in UIConfig itself? @colomolo has a better idea which `page` property was deleted

Reply:
Yep, found similar comment from @colomolo  here: https://github.com/contentful/doc-app/pull/2379

Reply:
I have that in the TODO list in the PR description 🤓

Reply:
will wait for approval then